camilo-alfonso-tarea3/oddEven.js

```diff
@@ -0,0 +1,13 @@
+/* Given a list of integers, determine whether the sum of its elements is odd or even.
+   Give your answer as a string matching "odd" or "even".
+   If the input array is empty consider it as: [0] (array with a zero). 
+*/
+
+function oddOrEven(array) {
+   let total = array.reduce((prev, curr) => prev+ curr, 0)
```

Comment:
@camosu , find a way to make this a one-line solution also use more than one return expression is a bad practice

Reply:
I have already corrected the exercises using ternary operator.